data/helpers.d/psql

```diff
@@ -0,0 +1,150 @@
+# Create a master password and set up global settings
+# Please always call this script in install and restore scripts
+#
+# usage: ynh_psql_test_if_first_run
+ynh_psql_test_if_first_run() {
+	if [ -f /etc/yunohost/psql ];
+	then
+		echo "PostgreSQL is already installed, no need to create master password"
+	else
+		pgsql=$(ynh_string_random)
+		pg_hba=""
+		echo "$pgsql" >> /etc/yunohost/psql
+
+		if [ -e /etc/postgresql/9.4/ ]
+		then
+			pg_hba=/etc/postgresql/9.4/main/pg_hba.conf
```

Comment:
Is the `/etc/postgresql/9.4/main/pg_hba.conf` file removed when migrating YunoHost along with PostgreSQL 9.6...?

Reply:
Yes I think so, I think I recall I checked this and the whole /etc/postgresql/9.4 folder gets removed so the current code should be fine